pip-instll.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Did you mean to call this file `pip-install`?

Reply:
Oops, yes.  Thanks!